src/groovy/org/transmartproject/rest/marshallers/HalOrJsonRenderer.groovy

```diff
@@ -100,6 +100,15 @@ class HalOrJsonRenderer<T> extends AbstractIncludeExcludeRenderer<T> implements
         }
     }
 
+    private String getContentType(String name) {
```

Comment:
Seems overkill, we only need that one method (and in simplified form, because we always have the encoding set). As for `CHARSET_ATTRIBUTE`, my first version actually found `"$name;charset=${this.encoding}"` more readable then `"${name}${CHARSET_ATTRIBUTE}${this.encoding}"`. YMMV